""" test exceptions. """


def convert(s):
    """ Convert to an integer."""
    x = int(s)
    return x


convert("33")

convert("test")
